setup.py

```diff
@@ -1,19 +1,22 @@
 from io import open
 from setuptools import setup
 
+with open('README.md') as read_me:
+    long_description = read_me.read()
```

Comment:
I can't remember why but there was some problem with reading this whole file

Reply:
5811203349aa550d5f07e9197831352e857c66bd PyPi wasn't rendering our markdown correctly. We opted for the single-line as a workaround, but I should probably spend the few extra minutes needed to just make it render nicely in both places.

Will keep this open as a reminder to do just that...

Thanks @andriyor